src/chunkserver/chunkserver.cpp

```diff
@@ -75,8 +90,127 @@ DEFINE_string(walFilePoolDir, "./0/", "WAL filepool location");
 DEFINE_string(walFilePoolMetaPath, "./walfilepool.meta",
                                     "WAL filepool meta path");
 
+DEFINE_string(pfs_cluster,
+    "",
+    "pfs cluster name, only for pfs");
+
+DEFINE_string(pfs_pbd_name,
+    "",
+    "pfs pbd name, only for pfs");
+DEFINE_int32(pfs_host_id,
+    -1,
+    "pfs host id, only for pfs");
+
 const char* kProtocalCurve = "curve";
 
+
+#ifdef WITH_SPDK
+void
+glog_pfs_func(int level, const char *file, const char *func, int line,
+    const char *fmt, va_list ap) {
+    char buf[8192];
+
+    int glevel = google::GLOG_INFO;
+    vsnprintf(buf, sizeof(buf), fmt, ap);
+    switch (level) {
+    case PFS_TRACE_FATAL:
+        glevel = google::GLOG_FATAL;
+        google::LogMessage(file, line, glevel).stream() << buf;
+        abort();
+        break;
+    case PFS_TRACE_ERROR:
+        glevel = google::GLOG_ERROR;
+        break;
+    case PFS_TRACE_WARN:
+        glevel = google::GLOG_WARNING;
+        break;
+    case PFS_TRACE_INFO:
+    case PFS_TRACE_DBG:
+    case PFS_TRACE_VERB:
+    default:
+        glevel = google::GLOG_INFO;
+        break;
+    }
+    google::LogMessage(file, line, glevel).stream() << buf;
+}
+
+void
+glog_spdk_func(int level, const char *a_file, const int a_line,
```

Comment:
A lot of glog_ related functions are implemented in chunkserver.cpp, but they are not used in this file. It is recommended to define them in a new file.

Reply:
done